emr-tablestore/src/main/scala/org/apache/spark/sql/aliyun/tablestore/TableStoreSink.scala

```diff
@@ -24,7 +24,9 @@ import org.apache.spark.sql.types.StructType
 class TableStoreSink(
     parameters: Map[String, String],
     userSpecifiedschema: Option[StructType]
-  )(@transient val sqlContext: SQLContext) extends Sink with Logging {
+)(@transient val sqlContext: SQLContext)
+    extends Sink
+    with Logging {
```

Comment:
minor: keep the previous code style here and other places

Reply:
done